platforms/common/uORB/uORBManager.hpp

```diff
@@ -353,9 +465,9 @@ class uORB::Manager
 
 	static uint8_t orb_get_instance(const void *node_handle);
 
-	static unsigned updates_available(const void *node_handle, unsigned last_generation) { return static_cast<const DeviceNode *>(node_handle)->updates_available(last_generation); }
+	static unsigned updates_available(const void *node_handle, unsigned last_generation);
 
-	static bool is_advertised(const void *node_handle) { return static_cast<const DeviceNode *>(node_handle)->is_advertised(); }
```

Comment:
Can you ifdef these to keep them inlined on flat builds?
I see ~1% load difference on stm32f4.

The rest looks good.

Reply:
Sure, will do that.